dev/reference/security/privilege-system.md

```diff
@@ -46,6 +46,8 @@ mysql> SELECT user,host FROM mysql.user WHERE user='xxxx';
 
 上述示例中，`xxxx@%` 即自动添加的用户。
 
+**Note:**
```

Comment:
Did you want to add a note format here? If yes, it's not complete and not correct.